sync_twindis/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Denis Mudarisov <https://github.com/trojikman>
+# License MIT (https://opensource.org/licenses/MIT).
+
+{
+    "name": """Sync Twindis""",
+    "summary": """Integration with Twindis' API""",
+    "category": "Extra Tools",
+    "images": ["images/sync_twindis.jpg"],
+    "version": "14.0.1.0.0",
+    "author": "IT-Projects Labs, Denis Mudarisov",
+    "support": "help@itpp.dev",
+    "website": "https://github.com/itpp-labs/sync-addons",
+    "license": "Other OSI approved licence",  # MIT
+    "depends": ["sale_management", "sync", "purchase"],
+    "external_dependencies": {"python": [], "bin": []},
+    "data": [
+        "data/sync_project_twindis_integration_data.xml",
```

Comment:
```suggestion
        "data/sync_project_data.xml",
```

Reply:
in fact, the module itself generates these types of file names
![Screenshot from 2021-07-03 23-34-41](https://user-images.githubusercontent.com/23641186/124363869-78177200-dc57-11eb-96d0-f4d55b3e05dd.png)